src/sbt-test/sbt-reactive-app/labels/build.sbt

```diff
@@ -20,6 +20,7 @@ TaskKey[Unit]("check") := {
   val outputDir = (stage in Docker).value
   val contents = IO.readLines(outputDir / "Dockerfile")
   val lines = Seq(
+    """ENTRYPOINT ["/rp-start", "bin/labels"]""",
```

Comment:
Don't we need to assert the Docker `ADD` command also?

Reply:
Good catch - added this.